platforms/software/testing-base/src/main/java/org/gradle/api/internal/tasks/testing/DefaultJUnitXmlReport.java

```diff
@@ -46,4 +50,14 @@ public void setOutputPerTestCase(boolean outputPerTestCase) {
     public Property<Boolean> getMergeReruns() {
         return mergeReruns;
     }
+
+    @Override
+    public Property<Boolean> getIncludeSystemOutLog() {
+        return includeSystemOutLog;
+    }
+
+    @Override
+    public Property<Boolean> getIncludeSystemErrLog() {
+        return includeSystemErrLog;
+    }
```

Comment:
I think you should be able to get away with abstract getters here now and you don't have to have the fields.